### PR TITLE
rename $<config> to <config>

### PR DESCRIPTION
**Pull request by moritzraho**
**Relates to adobe/aio-cli#191**
**Depends on adobe/aio-lib-ims#48**
**Relates to https://github.com/adobe/aio-lib-ims/issues/47**

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
   "bugs": "https://github.com/adobe/aio-cli-plugin-app/issues",
   "dependencies": {
     "@adobe/aio-app-scripts": "^2.0.0",
-    "@adobe/aio-lib-core-config": ">=1.2.8",
+    "@adobe/aio-lib-core-config": "2.0.0",
```
Comment (shazron) [inline package.json:9]: should be caret? we want to pick up any path or minor releases as well

File: test/commands/lib/import.test.js [hunk 2]
```diff
@@ -277,7 +277,7 @@ test('enrich $ims jwt credential with project.org.ims_org_id', async () => {
   expect(fs.writeFile).toHaveBeenCalledTimes(2)
 })
 
-test('do not enrich $ims.jwt with ims_org_id if no jwt credentials defined ', async () => {
+test('do not enrich ims.jwt with ims_org_id if no jwt credentials defined ', async () => {
```
Comment (shazron) [inline test/commands/lib/import.test.js:280]: nitpick: ims.contexts.jwt (or make it consistent with the test above :)

File: package.json (hunk 1 above)
Comment (moritzraho) [reply package.json:9]: +1 my bad

File: test/commands/lib/import.test.js (hunk 2 above)
Comment (moritzraho) [reply test/commands/lib/import.test.js:280]: ok makes sense

**Pull request merged**